packages/core/src/bridge.ts

```diff
@@ -112,6 +112,13 @@ export const useWorldBridgeStore = create<WorldBridgeStore>((set, get) => ({
 
 		const res = await fetch(`${get().bridge_url}/response/${get().requestId}`)
 
+		if (!res.ok) {
```

Comment:
this is fine for now as it's already an improvement, but we should handle more explicitly here:
1. if the request returns a 404 (after previously having returned a 200) it means the whole verification request expired, we should show a specific error for this.
2. if the request returns a 429 it means rate limits are being hit, we should retry with exponential backoff and let the user know this will take several seconds longer.
3. any other error (particularly 5xx) we should silently retry 2-3 times before showing a final error to the user

Reply:
created a linear issue for these further improvements: https://linear.app/worldcoin/issue/WID-598/improve-bridge-status-handling-in-idkit-core

resolving for now